server-render/index.ts

```diff
@@ -0,0 +1,12 @@
+import {parse, print} from './ast';
+import {renderAmpAst} from './render';
+
+/*
+ * Renders all of the eligible AMP Components within an AMP Document
+ */
+export function renderAmpHtml(html: string): string {
```

Comment:
nit: shouldn't this live in `./render` too?

Reply:
Done